Tools/parametric_model/src/tools/dataframe_tools.py

```diff
@@ -46,46 +46,59 @@
 def compute_flight_time(data_df):
     """
     The flight time will be determined based on the 'landed' topic of the ulog to only consider actual flight during identification.
+
+    Depending on the number of sections where the landed topic is 1, the following cases are distinguished:
+    - No groups with landed = 1: No flight detected. Start and end time of the flight will be returned as zero
+    - One group with landed = 1: The start and end time of the entire flight log will be returned
+    - Even number of groups with landed = 1: An array with start and end time of each detected flight segment will be returned. If multiple flight segments were detected, a warning is issued.
+    - Odd number of groups with landed = 1: An array with start and end time of each detected flight segment will be returned. If multiple flight segments were detected, a warning is issued.
     """
     print("\nComputing flight time...")
     landed_groups = data_df.groupby(
         (data_df["landed"].shift() != data_df["landed"]).cumsum()
     )
     num_of_groups = landed_groups.ngroups
 
+    # single group detected - flight either started and ended outside of log of aircraft did not fly at all
     if num_of_groups == 1:
```

Comment:
A minor comment here. I was just thinking that the functionality of what num_of_groups is exactly doing is not intuitively very clear to the reader and it took me a while again to wrap my head around that. maybe we could improve the readability by creating an enum for each of these cases (e.g. SINGLE_FLIGHT_DETECTED = 1 ) and use a switch case systems instead of the many if statements. 

This is purely a style suggestion though, not a hard requirement :)

Reply:
I think, this would be a good idea, if we decide that we need a more complicated logic here at some point. For now, I would say that the definition of an enum with only two cases and a switch statement is not that much more intuitive compared to a simple if-else statement with a comment?
But it is definitely a good idea if this part would be extended further :)